app/views/shared/_search_form.html.erb

```diff
@@ -1,3 +1,30 @@
+
```

Comment:
The alignment of the `Find` button is no longer aligned properly
<img width="325" alt="screen shot 2017-07-27 at 9 01 19 am" src="https://user-images.githubusercontent.com/174744/28671332-459b780a-72aa-11e7-83c8-3dfaf9658258.png">